src/app/routes/article/getInitialData/index.test.ts

```diff
@@ -292,4 +294,32 @@ describe('Articles - BFF Fetching', () => {
       status: 500,
     });
   });
+
+  it('should transform response as expected', async () => {
+    const fetchDataSpy = jest.spyOn(fetchPageData, 'default');
+
+    fetchDataSpy.mockImplementation(() =>
+      Promise.resolve({
+        status: 200,
+        json: pidginArticleWithLatestMedia,
+      }),
+    );
+
+    const { pageData } = await getInitialData({
```

Comment:
I cannot for the life of me figure out why this is not working - @andrewscfc or @amoore108, do you have any ideas? I don't reallllly want to use `ts-expect-error`

Reply:
It could be because TS doesn't know if `getInitialData` resolves or rejects, so you may need to cast the expected value like:
```
   const { pageData } = (await getInitialData({
      path: '/kyrgyz/articles/c0000000000o',
      getAgent,
      service: 'kyrgyz',
      pageType: 'article',
    })) as { pageData: Record<string, unknown> };
```